.circleci/test-deploy.yml

```diff
@@ -8,7 +8,9 @@ filters: &filters
 
 jobs:
   install-and-persist:
-    executor: cypress/default
+    executor:
+      name: cypress/default
+      node-version: "20.6.0"
```

Comment:
should we use `18.16.1` to cover what is in the default config or is that covered elsewhere? Sorry I'm a bit unfamiliar 🥴

Reply:
Well, I wanted to use a different version than the default to validate it works with a different version than the default

Reply:
that makes sense. I was just wondering where we test the `default`.